docs/development/i18n.rst

```diff
@@ -92,43 +100,58 @@ new language is completely translated and reviewed, the
 ``i18n_tool.py`` file must be manually edited to add this new language
 to the ``SUPPORTED_LANGUAGES`` variable.
 
+.. _update_strings_to_be_translated:
+
 Update strings to be translated
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-After strings are modified in the SecureDrop source code, templates or
-desktop labels, the ``securedrop/translations/messages.pot`` files
-must also be updated. Individual developers should not do this
-whenever changing strings in the code; the translations are updated in
-bulk when it's time to update the Weblate fork.
-
-Translations can be updated with the following command:
-
-.. code:: sh
-
-    $ make translate
-
-This wraps ``i18n_tool.py translate-messages`` and ``i18n_tool.py
-translate-desktop``.  These commands update the `.pot`_ files for the
-SecureDrop server code and the desktop icons, as well as the `.po`_
-files for each language.
-
-.. note:: The new source strings will only be visible to translators
-    in `Weblate`_ after the ``develop`` branch is merged into the
-    Weblate fork.
+Whenever strings are modified in the SecureDrop source code, templates
+or desktop icon labels, the translation files should also be updated
+by running ``make translate`` in the root of the SecureDrop working
+copy.
+
+The ``translate`` target runs ``i18n_tool.py translate-messages`` and
+``i18n_tool.py translate-desktop``, which in turn use ``pybabel
+extract`` to gather source strings.  These commands update the `.pot`_
+files for the SecureDrop server code and the desktop icons, as well as
+the `.po`_ files for each language.
+
+After running ``make translate``, carefully review the output of ``git
+diff``. Check ``securedrop/messages.pot`` first for updated strings,
+looking for problems like:
+
+* overly idiomatic English
+* fragmented text, such as pieces of a sentence intended to be
+  concatenated together, which can be difficult to translate
+* messages that are marked with plain ``gettext`` and contain plurals
+  based on numeric placeholder variables -- these should generally be
+  marked with ``ngettext`` so that they can be translated properly in
+  languages with complex plural forms
+
+Then review the ``messages.po`` of one existing translation, with a
+focus on new translations, which are often marked ``fuzzy``. There is
+no need to review multiple languages' ``.po`` files because they are
+processed in the same way.
```

Comment:
so is the developer just confirming that all the expected strings that chnaged appeared in the file? What does "fuzzy" mean?

Reply:
I added a link to the [gettext docs on "fuzzy"](https://www.gnu.org/software/gettext/manual/html_node/Fuzzy-Entries.html).